sql-gremlin/src/test/java/org/twilmes/sql/gremlin/adapter/GremlinSqlAdvancedSelectTest.java

```diff
@@ -41,6 +41,32 @@ public void testProject() throws SQLException {
                 rows(r("Tom"), r("Patty"), r("Phil"), r("Susan"), r("Juanita"), r("Pavel")));
     }
 
+    @Test
+    public void testOrder() throws SQLException {
```

Comment:
Can we add a unit test (not integration test) to compare that translation to gremlin is correct?

Reply:
With the current setup, it's technically possible.... We need to send in SQL to get our syntax tree out, and the Gremlin that comes out is actually a GraphTraversal Object and is a pretty complex traversal, doing it would ultimately be a really hacky setup.